google/cloud/spanner_dbapi/connection.py

```diff
@@ -486,22 +529,28 @@ def execute_batch_dml_statement(self, parsed_statement: ParsedStatement):
 
     @check_not_closed
     def run_batch(self):
+        result_set = None
```

Comment:
I think that we should consider returning something in case of a DDL batch, for example the number of DDL statements that was executed. In Java, we return an array that has the same length as the number of statements, and that contains a 1 for each statement that we executed. This specific return type has to do with that JDBC expects a driver to return, and in this case I think we have some more freedom. What do other database systems return in case like this? For example if you in PostgreSQL execute something like the following SQL string: `create table table1 (...); create table table2 (...); create index index1 on table1 (...);`

Reply:
Tested that postgres also returns None

Reply:
Ack. Let's keep it None here as well then 👍